src/applications/personalization/profile360/components/Address.jsx

```diff
@@ -6,7 +6,7 @@ import { focusElement } from '../../../../platform/utilities/ui';
 
 import ErrorableSelect from '../../../letters/components/ErrorableSelect';
 import ErrorableTextInput from '../../../letters/components/ErrorableTextInput';
-import { STATE_CODE_TO_NAME, MILITARY_CITIES } from '../../../letters/utils/constants';
+import { STATE_CODE_TO_NAME, MILITARY_CITIES, MILITARY_STATES } from '../../../letters/utils/constants';
 import { militaryStateNames } from '../../../letters/utils/helpers';
```

Comment:
I'm not sure if it's changed since, but I was trying to centralize the address work into `platform/forms/address`. Not sure if it has what you need in there.